.github/ISSUE_TEMPLATE/b-bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "🐛 Bug Report"
+about: Found a bug? Click here and let us know!
+labels: bug
+---
+
+<!-- Please give the issue a short, clear title -->
+
+## 🐛 Bug Report
+<!-- Summarise the bug here as clearly and concisely as you can -->
```

Comment:
nit: leave empty line below markdown header. You do this for the question template

Reply:
I removed the empty line in the question template since it was the only place that had it. I see no reason for needing it.

Reply:
Markdownlint [1] suggests blank lines around the header on both sides (which is what I personally use myself), but doesn't provide a particularly strong argument. I would prefer the blank lines, but this is mostly for aesthetic reasons, so it is not a super strong opinion.

[1] https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines

Reply:
Under normal circumstances, I would empty lines but I think in the case of these comments it's fine. (I would also leave two empty lines above ## (h2) for better separation, but I don't think that does any good here.